src/main/java/com/firstwebapp/practice/spring/Album.java

```diff
@@ -0,0 +1,19 @@
+package com.firstwebapp.practice.spring;
+
+
+public class Album {

```

Comment:
It's good that you have this class, but it isn't set up to interact with a database at all.